streaming/src/main/scala/org/apache/spark/streaming/scheduler/StreamingListener.scala

```diff
@@ -66,6 +69,9 @@ case class StreamingListenerReceiverStopped(receiverInfo: ReceiverInfo)
 @DeveloperApi
 trait StreamingListener {
 
+  /** Called when the streaming starts */
```

Comment:
This doesn't sound accurate. This is the time when the `StreamingContext` was created, right?

Of if this is really what you mean, you need to fix the code in `StreamingContext.scala`.